### DISCUSSION TITLE
About _event question

### DISCUSSION THREAD

**Discussion by xiaoxiji1234** [lowdefy/lowdefy#1054] [Questions and Help]
Hi, Can I use it The _event operator obtains the return value in the function in the custom block?
For example, the following functions exist in a custom block:
```js
function getData(data){
return data
}
```
Can I use it directly in lowdefy _event operator  gets the return value of GetData?

**Comment by SamTolmay** [comment]
Hi @xiaoxiji1234 

You can use the only use `_event` operator in actions. The actions then have access to the event passed by the block. You can pass the event data as a parameter when calling the `triggerEvent` method. So you would do:

```js
methods.triggerEvent({
    name: 'onData',
    event: data, // or event: { data },
});
```
